### PR TITLE
fix(nl): change informal to formal text for consitency

### PR DESCRIPTION
**Pull request by ricardovanlaarhoven**
# Explain your changes

change informal to formal text for consitency](https://github.com/kinde-oss/kinde-translations/commit/3e3a08308d065a5fed9b8330367479cddcf5901f)

# Checklist

- [x] I have read the [“Pull requests” section](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md#pull-requests) in the [contributing guidelines](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/main/.github/CODE_OF_CONDUCT.md).

🛟 _If you need help, consider asking for advice over in the [Kinde community](https://thekindecommunity.slack.com)._

### CODE REVIEW

**Comment by DaveOrDead** [comment]
Thanks @ricardovanlaarhoven - apologies for the delay on this one. It seems like the validator didn't pass :(

**Comment by ricardovanlaarhoven** [comment]
@DaveOrDead, thats weird, since theres only a textual change right?

**Comment by DaveOrDead** [comment]
@ricardovanlaarhoven the validator runs against the whole schema so probably there are other missing translations for NL: https://github.com/kinde-oss/kinde-translations/actions/runs/10299004975/job/29283549683?pr=85

File: nl/auth.json [hunk 1]
```diff
@@ -280,5 +280,69 @@
     "verification_code_sms": {
       "description": "${generated_code} is uw eenmalige code om u aan te melden bij ${business_name}\n\n@${receiver_domain} #${generated_code}"
     }
```
Comment (ricardovanlaarhoven) [inline nl/auth.json:282]:
```suggestion
    },
```

**Review by DaveOrDead — APPROVED** [review]

**Comment by DaveOrDead** [comment]
thanks @ricardovanlaarhoven ! 🚀

**Pull request merged**